### DISCUSSION TITLE
Marketing en la práctica CSS

### DISCUSSION THREAD

**Discussion by GloriaCoralCerecedo** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#382] [Q&A]
Buenos días, ¿Donde hicieron la campaña de marketing?

**Comment by KevinRomero2** [comment]
Buenos Días, yo planeo hacerlo en miro, pero puede servir cualquier página  o aplicación para hacer diagramas, esquemas, ya es más que nada planificar la campaña de forma de saber como la vamos a hacer y como vamos a distribuir la información.

**Comment by GloriaCoralCerecedo** [comment]
Oh vale osea no hay que poner mucha info solo es como por ejemplo 

Realizar una campaña en redes sociales, publicar imágenes y enlaces a la página 

¿Algo así? 

No sé si se podrá manejar una tabla o como un calendario.

Igual la práctica de css dijo que el MC que la aplazara un poco más no(?)

**Reply by martinGM05** [reply]
Pues basicamente es un plan de mercadotecnia, solo es como dices una tabla donde pongas que actividades, que contenido, etc.. y pues un calendario podria.

Y si, de hecho es para el 20 de este mes.

**Reply by GloriaCoralCerecedo** [reply]
Oh vale gracias

**Comment by Ingrzs** [comment]
Pues yo la hice en miro en una tabla
Checa mi repo en donde dice Campaña de Marketing(esta en pdf)
https://github.com/Ingrzs/landing-vacunacion

**Comment by GloriaCoralCerecedo** [comment]
Muchas gracias, igual vi tu página te quedo genial

**Comment by JuanSalvador73** [comment]
Yo estoy usando miro y probablemente Illustrator. Realmente no necesariamente tiene que ser un documento de texto, puede ser un esquema como mencionan arriba otros explorers.

Conque sea conciso, claro y comprensible es mas que suficiente. Es como lo que hicimos con la práctica de los abogados al inicio de todo esto.

**Reply by GloriaCoralCerecedo** [reply]
Gracias
